activation/atxdb.go

```diff
@@ -110,10 +109,6 @@ func (db *ActivationDb) CalcActiveSetFromView(a *types.ActivationTx) (uint32, er
 			if err != nil {
 				return fmt.Errorf("error fetching atx %x from database -- inconsistent state", id)
```

Comment:
this error message should state that either the state is inconsistent or that the atx is invalid

Reply:
should we exit traversing in this case?

Reply:
If we have the ATX ID in the db we should also have the ATX. If this happened it's a serious error that should never happen. I actually think that exiting with an error is not enough and we should panic, but somewhere else there was a discussion about this point and someone told me that exiting with an error is enough in such a case (I don't remember who, maybe @zalmen?).

Reply:
Replace with panic. Also, print a short id instead of the full id

Reply:
done (also in other places)